Modules/_ssl.c

```diff
@@ -138,10 +138,10 @@ struct py_ssl_library_code {
 #else /* OpenSSL < 1.1.0 */
 #if defined(WITH_THREAD)
 #define HAVE_OPENSSL_CRYPTO_LOCK
-/* For some reason, this function is not declared on OpenSSL's headers */
-void OPENSSL_cpuid_setup(void);
 #endif
 
+/* For some reason, this function is not declared on OpenSSL's headers */
+void OPENSSL_cpuid_setup(void);
```

Comment:
Is this function guaranteed to be available on all versions of OpenSSL and LibreSSL that are supported by Python? At the moment it's 0.9.8 to 1.1.x and all versions of LibreSSL?